Glycine/max/wip_marlene/Reid_Ferguson_2011

```diff
@@ -0,0 +1,68 @@
+gene_symbols:
+  - RIC1
+  - GmRIC1
+gene_symbol_long: rhizobia-induced CLE
+gene_model_pub_name: Glyma.13g36830
+gene_model_full_id: glyma.Wm82.gnm2.ann1.Glyma.13G368300
+confidence: 5
+comments:
+  - RT-qPCR studies conducted on tissue injected with Bradyrhizobium nodulation factor showed that RIC1 is upregulated hours after injection.
+  - Introduction of plasmids carrying an extra dose of RIC1 inhibited nodulation.
+phenotype_synopsis: RIC1 reduces nodulation in Lotus japonicus and Medicago truncatula. It is induced by Bradyrhizobium nodulation factor.
+traits:
+  - entity_name: plant organ growth and development trait
+    entity: TO:0000927
+  - entity_name: root nodule morphology trait
+    entity: TO:0000898
+  - entity_name: root nodule
+    entity: PO:0003023
+references:
+  - citation: Reid, Ferguson et al., 2011
+    doi: 10.1094/MPMI-09-10-0207
+    pmid: 21198362
+ ---
+ gene_symbols:
+  - RIC2
+  - GmRIC2
+gene_symbol_long: rhizobia-induced CLE
+gene_model_pub_name: Glyma.06g43680
+gene_model_full_id: NOT FOUND
```

Comment:
I'm having trouble finding the gene ID for Glyma.06g43680.

Reply:
I searched for GmRIC1 in the ncbi database and found a different pub name than the one in the paper: Glyma.13G292300.  It sounds like the same gene based on the description and it does have an ID: glyma.Wm82.gnm2.ann1.Glyma.13G292300